bedrock/firefox/templates/firefox/firstrun/index.html

```diff
@@ -13,7 +13,7 @@
 {% endblock %}
 
 {% block site_css %}
-  {% stylesheet 'firefox_firstrun_onboarding' %}
+  {% stylesheet 'firefox_firstrun' %}
 {% endblock %}
 
```

Comment:
I see one block didn't get copied over *coughoptimizelycough*. I'm not mad about it, however it does still exist in the `new/base.html` template. Might be worth putting in `firstrun/base.html` for future requests?

Reply:
Not directly related to this PR, but wondering if we should implement the open graph stuff (`og:url` in particular) from the old `firstrun-horizon.html` template over to `firstrun/base.html`?

Reply:
I'll take a look at this 👍

Reply:
I've added optimizely to `/firstrun/base.html`.

As for the open graph strings, I do agree it makes sense for them to belong in `firstrun/base.html`, so we don't have to duplicate them for every page. This does mean adding `{% add_lang_files "firefox/new/onboarding" %}` to the base template though, so I want to make sure @flodolo is ok with the change.

Reply:
Definitely not OK, we're removing that file. Please add the strings behind a tag, we'll import them if they're already translated in the old file.

Reply:
Ok! 👍

Reply:
I should not comment while I have two feet out of the door (sorry, complicated period…).

`{% add_lang_files "firefox/new/onboarding" %}` is fine. I've thought that you were adding the old template (horizon).